### PR TITLE
Ensure popup closes automatically on user selection

### PR DESCRIPTION
**Pull request by banobepascal**
Change-Id: I4e7a06bab413dad0b5f2360c4bef729db7d5e0d0

* Resolves: #10167 
* Target version: master 

### Summary
Fixes a UX bug that disrupts user workflow by leaving the selection popup open after a user is chosen, blocking underlying content.

### TODO
Users Clicks in the avatars
Pop up closes when one of the users is clicked 

### Checklist

- ✅ I have run `make prettier-write` and formatted the code.
- ✅  All commits have Change-Id
- ✅  I have run tests with `make check`
- ✅ I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by pedropintosilva** [comment]
Thanks @banobepascal @eszkadev and I will take a look. Meanwhile could you please fix the DCO check but following the instructions at: image

**Comment by banobepascal** [comment]
> Thanks @banobepascal @eszkadev and I will take a look. Meanwhile could you please fix the DCO check but following the instructions at: image

Hey @pedropintosilva, I'm working on this now

File: browser/src/control/Control.UserList.ts [hunk 1]
```diff
@@ -520,6 +520,7 @@ class UserList extends L.Control {
 			listItem.appendChild(userLabelContainer);
 			listItem.addEventListener('click', () => {
 				this.followUser(viewId);
+				JSDialog.CloseDropdown('userlist');
```
Comment (eszkadev) [inline browser/src/control/Control.UserList.ts:523]: looks good :)

**Review by eszkadev — APPROVED** [review]

**Comment by pedropintosilva** [comment]
it seems the failing test is integration_tests/multiuser/calc/cell_cursor_spec.js which might be a false negative . Ideally you would re-run the test locally to see if it really fails due to your change or if it's the test that is unstable (if it's unstable we can just press the button retry by: clicking the details url and then log in with your Collabora credentials)    

paste from the CI logs:

```javascxript

**Comment by pedropintosilva** [comment]
meanwhile I have retried the CI to see if it keeps failing, but please @banobepascal check the test locally

**Comment by banobepascal** [comment]
> * make -C cypress_test run-multi spec=calc/cell_cursor_spec.js

@pedropintosilva I'm running the tests locally, I'm going to provide an update

**Comment by eszkadev** [comment]
This test seems to be related `.jsdialog-overlay` is used for popups. Here we close popup which wasn;t closed before. It might be expected by the test (what should be modified now)

**Comment by banobepascal** [comment]
> This test seems to be related `.jsdialog-overlay` is used for popups. Here we close popup which wasn;t closed before. It might be expected by the test (what should be modified now)

We can add an explicit wait or assertion to ensure the popup is visible before attempting to close it as this will give the application time to render the popup before the test interacts with it.

**Comment by pedropintosilva** [comment]
So, that line 
```javascript
		cy.cGet('.jsdialog-overlay').click({force: true});
```

that is in that test twice should just be removed.

Alternatively we would assert that '.jsdialog-overlay' doesn't exist thus, introducing a test for your improvement via `.should('not.exist');`

**Pull request merged**